mozilla/blurts-server
# Explain nuance of deleting account with Plus

# References: 
Jira: MNTOR-3035
Figma: https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40/Concepts-for-Monitor-MVP-Redesign?type=design&node-id=8356%3A64580&mode=design&t=n7I3bAVC6wfZNo57-1



# Description

When someone has a Plus subscription and deletes their Monitor account (without deleting their Mozilla account), then signing in again to Monitor will re-create the account and re-activate the Plus features if their terminated subscription hasn't lapsed yet.

# Screenshot (if applicable)

![image](https://github.com/mozilla/blurts-server/assets/4251/0618131a-ff5f-439c-a121-b821a85996a3)

# How to test

When signed in with a Plus user and with the `MonitorAccountDeletion` flag enabled, visit the settings page, and click the "Delete account" link. You should see the new text in the dialog.

# Checklist (Definition of Done)
